code/modules/lootpanel/ss_looting.dm

```diff
@@ -2,8 +2,9 @@
 /// Queues image generation for search objects without icons
 SUBSYSTEM_DEF(looting)
 	name = "Loot Icon Generation"
-	init_order = INIT_ORDER_LOOT
+	flags = SS_NO_INIT
```

Comment:
(we should also get rid of the `INIT_ORDER_LOOT` define if it's completely unused now)